### PR TITLE
Remove elastic resorces rewrite

### PR DESCRIPTION
**Pull request by yotamazriel**

### CODE REVIEW

File: constructs/elasticsearch.ts [hunk 1]
```diff
@@ -56,16 +56,6 @@ curl 127.0.0.1:9200/_template/zero-replicas -s --fail -XPUT -H 'Content-Type: ap
         },
         antiAffinity: 'soft',
         esJavaOpts: '-Xmx128m -Xms128m -Dlog4j2.formatMsgNoLookups=true',
```
Comment (arikmaor) [inline constructs/elasticsearch.ts:58]: You also need to change this environment variables

Comment (yotamazriel) [reply constructs/elasticsearch.ts:58]: Thanks!

File: main.ts [hunk 2]
```diff
@@ -17,11 +17,11 @@ new Kibana(app);
 const minio = new Minio(app);
 new RabbitMQ(app);
 new NodeServer(app, {
-  imageTag: 'master-d2d092f4-stable',
+  imageTag: 'master-2aef48b9-stable',
   minioAddress: minio.minioAddress,
 });
 new Engine(app, {
-  imageTag: 'master-9108dbba-stable',
+  imageTag: 'remove-es-refresh-a88df799',
```
Comment (doronHarnoy) [inline main.ts:24]: should we switch this version to master?

**Review by doronHarnoy — APPROVED** [review]
LGTM

**Review by doronHarnoy — APPROVED** [review]
LGTM

File: __snapshots__/main.test.ts.snap [hunk 3]
```diff
@@ -129,7 +129,7 @@ spec:
             - name: network.host
               value: 0.0.0.0
             - name: ES_JAVA_OPTS
-              value: -Xmx128m -Xms128m -Dlog4j2.formatMsgNoLookups=true
+              value: -Dlog4j2.formatMsgNoLookups=true
```
Comment (arikmaor) [inline __snapshots__/main.test.ts.snap:132]: I'm not sure you should have removed it
It is set by default to something else

**Pull request merged**